diff -Naur QtCurve-KDE3-1.3.0-orig/config/qtcurveconfig.cpp QtCurve-KDE3-1.3.0/config/qtcurveconfig.cpp
--- QtCurve-KDE3-1.3.0-orig/config/qtcurveconfig.cpp	2010-04-11 23:59:59.000000000 +0100
+++ QtCurve-KDE3-1.3.0/config/qtcurveconfig.cpp	2010-04-12 18:07:15.000000000 +0100
@@ -49,6 +49,8 @@
 #include <kguiitem.h>
 #include <kinputdialog.h>
 #include <knuminput.h>
+#include <kglobal.h>
+#include <kstandarddirs.h>
 #include <unistd.h>
 #include <errno.h>
 #include "config.h"
